plugin-tester-java/build.gradle

```diff
@@ -27,7 +27,7 @@ akkaGrpc {
   generateClient = true
   generateServer = true
   serverPowerApis = true // This isn't default
-  extraGenerators = [ 'akka.grpc.gen.scaladsl.play.PlayScalaClientCodeGenerator', 'akka.grpc.gen.scaladsl.play.PlayScalaServerCodeGenerator' ]
+  extraGenerators = [ ]
```

Comment:
How is this related to this PR? Does it also remove play modules, is it on top of the remove play modules-pr?

Reply:
Apparently, this is built on top of #598. So maybe @raboof need to do a rebase here?

Reply:
(:+1: rebased for smaller diff)